src/AuthenticatedAPIClient/tests/AuthenticatedAPIClient.test.jsx

```diff
@@ -152,7 +157,7 @@ describe('AuthenticatedAPIClient auth interface', () => {
       const expectedRedirectUrl = encodeURIComponent(process.env.BASE_URL);
       const expectedLocation = `${loginUrl}?next=${expectedRedirectUrl}`;
       // eslint-disable-next-line prefer-promise-reject-errors
-      mockAccessToken.get.mockReturnValue(Promise.resolve(emptyUserAccessToken));
+      mockAccessToken.get.mockReturnValue(Promise.resolve(null));
```

Comment:
I'd like to see the final PR once merged.  Are we changing the interface, or just going back to what it used to do?